doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+
+Validation of the input file is automatically handled by the NEAMS Workbench GUI, and validation errors are shown in the `Validation` tab highlighted in red and located at the bottom right of the GUI, see [fig:neams_workbench_ncrc_if]. It should be noted that some of the validation errors are due to some syntax not being supported by the NEAMS Workbench. These issues are being addressed and will be fixed in a future release of the NEAMS Workbench GUI. These particular validation errors can be ignored for now and will not prevent the job from running properly (see Section 5 for further details). By clicking on the validation error message, the user is taken to the line with the syntax error. Auto-completion is also available to help with file editing, which is illustrated at time 1:41 in the *[demonstration video](https://www.youtube.com/watch?v=W6Rq_owHmoU&ab_channel=MarcoDelchini)*.
+
+!media neams_workbench/neams_workbench_ncrc_if.png style=width:80% id=fig:neams_workbench_ncrc_if caption=Input file (left) and mesh visualization (right) with the NEAMS Workbench.
+
+### Step 3: running a job from the NEAMS Workbench GUI
+
+After editing the input file and making sure that the physics enabled are consistent with the geometry visualized with ParaView, the input file is submitted to the scheduler. The NEAMS Workbench has already been set to run this input file with the _BlueCRAB_ executable (configured in Step 1). To submit the job to `Sawtooth`, simply type `Run`, which will create the required job scheduler script behind the scenes and put your job in the queue. To customize your job run options first, click the downward arrow next to `Run` and select `Customize Run Options...`. A widget opens that allows users to modify the number of cores and CPU time by scrolling down to `Remote: Scheduler Header` and modifying the number of nodes (select), numbers of CPUs per node (ncpus) and walltime (walltime), as illustrated in [fig:hpc_on_demand_pbs_options]. The suggested number of nodes/CPUs and walltime should be given in the VTB documentation for each test case. Progress of the job is displayed in the `Message`: the NEAMS Workbench checks on the status of the job and tail the output once it starts to run on Sawtooth. Once the job completes, it will display `Processed Finished`.
```

Comment:
```suggestion
After editing the input file and making sure that the physics enabled are consistent with the geometry visualized with ParaView, the input file is submitted to the scheduler. The NEAMS Workbench has already been set to run this input file with the _BlueCRAB_ executable (configured in Step 1). To submit the job to `Sawtooth`, simply type `Run`, which will create the required job scheduler script behind the scenes and put your job in the queue. To customize your job run options first, click the downward arrow next to `Run` and select `Customize Run Options...`. A widget opens that allows users to modify the number of cores and CPU time by scrolling down to `Remote: Scheduler Header` and modifying the number of nodes (select), numbers of CPUs per node (ncpus) and walltime (walltime), as illustrated in [fig:hpc_on_demand_pbs_options]. The suggested number of nodes/CPUs and walltime should be given in the VTB documentation for each test case. Progress of the job is displayed in the `Message`: the NEAMS Workbench checks on the status of the job and tail the output once it starts to run on Sawtooth. Once the job completes, it will display `Process Finished`.
```